### PR TITLE
V1.83 cherry picks

### PR DESCRIPTION
**Pull request by rgroemmer**
/kind enhancement
/override pull-gardener-e2e-kind

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-v1.83.3
\ No newline at end of file
+v1.83.3-ske
```
Comment (MichaelEischer) [inline VERSION:1]: The version should be `v1.83.3-ske-1`

**Comment by Duciwuci** [comment]
You are right! I removed my lgtm..

**Comment by Duciwuci** [comment]
Thanks, Dr.!

**Comment by Duciwuci** [comment]
https://github.com/stackitcloud/ske-ci-infra/pull/887 this PR will fix your issue.

**Pull request merged**